app/controllers/api/admins_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::AdminsController < Api::BaseController
+  def reset_password
+    @admin = Admin.find_by_email(reset_params)
+    if @admin.present?
+      @admin.send_reset_password_instructions
+      success_response('You will receive your email shortly!')
```

Comment:
nit - can we reword to something like "You will receive an email shortly with instructions" or something along that vein?